spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -1959,9 +1959,19 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
 
         case UnaryMinus(child, _) =>
```

Comment:
Hmmm, should we use the second member `failOnError` instead of reading `SQLConf.get.ansiEnabled`?

Looks like `EvalMode::Try` is not really used, if I understand correctly.

Reply:
This was nice, thanks.